war/src/main/webapp/scripts/sortable.js

```diff
@@ -266,6 +270,35 @@ var Sortable = (function() {
     arrowTable.down.next = arrowTable.up;
 
 
+    function alphanum_sort(a,b,caseInsensitive) {
+        function chunkify(t) {
+          var tz = [];
+          var x = 0, y = -1, n = 0, i, j;
+
+          while (i = (j = t.charAt(x++)).charCodeAt(0)) {
+            var m = (i == 46 || (i >=48 && i <= 57));
+            if (m !== n) {
+              tz[++y] = "";
+              n = m;
+            }
+            tz[y] += j;
+          }
+          return tz;
+        }
+
+        var aa = chunkify(caseInsensitive ? a.toLowerCase() : a);
+        var bb = chunkify(caseInsensitive ? b.toLowerCase() : b);
+
+        for (x = 0; aa[x] && bb[x]; x++) {
+          if (aa[x] !== bb[x]) {
+            var c = Number(aa[x]), d = Number(bb[x]);
+            if (c == aa[x] && d == bb[x]) {
+              return c - d;
+            } else return (aa[x] > bb[x]) ? 1 : -1;
+          }
+        }
+        return aa.length - bb.length;
+    }
```

Comment:
Yes, that code is mainly based on the http://www.davekoelle.com/files/alphanum.js which is LGPL. Based on https://softwareengineering.stackexchange.com/questions/141847/how-does-using-a-lgpl-gem-affect-my-mit-licensed-application i would assume that there is no license conflict.

Please let me know if you want to make any changes to the code as we would all benefit from merging this.

The code worked for me well in Firefox, Chrome and Safari. I didn't had any other browsers to test with but based on its nature I doubt we would encounter any browser specific issues.